src/test/resources/mappings/authn.json

```diff
@@ -12,8 +12,8 @@
           "Set-Cookie": ["folioAccessToken=accessToken","folioRefreshToken=refreshToken"]
         },
         "jsonBody": {
-          "accessTokenExpiration":  "{{now offset='1 days'}}",
-          "refreshTokenExpiration": "{{now offset='1 days'}}"
+          "accessTokenExpiration":  "2024-03-15T12:00:00Z",
```

Comment:
Will it work after 15 march?
Maybe it should be properly modified?
https://wiremock.org/docs/response-templating/#date-and-time-helpers

Reply:
@psmagin **parseExpiration** in the **TokenUtils** method is responsible for parsing, but it accepts the String which cannot be properly interpreted, one of the ways is to change the date to longer one or change the parseExpiration method, which one would be appropriate?